### PR TITLE
Add DevFest Venice 2024

### PR DESCRIPTION
**Pull request by prof18**

### CODE REVIEW

**Review by martinbonnin — APPROVED** [review]

File: backend/service-import/src/jvmMain/kotlin/dev/johnoreilly/confetti/backend/import/Sessionize.kt [hunk 1]
```diff
@@ -695,6 +695,30 @@ object Sessionize {
         )
     }
 
+    suspend fun importDevFestVenice2024(): Int {
+        return writeData(
+            getData("https://sessionize.com/api/v2/r14kt8fj/view/All"),
+            config = DConfig(
+                id = ConferenceId.DevFestVenice2023.id,
```
Comment (martinbonnin) [inline backend/service-import/src/jvmMain/kotlin/dev/johnoreilly/confetti/backend/import/Sessionize.kt:702]: Woops, I missed that one, this just overwrote the 2023 data. I'll make a patch quickly

Comment (martinbonnin) [reply backend/service-import/src/jvmMain/kotlin/dev/johnoreilly/confetti/backend/import/Sessionize.kt:702]: Should be all good now

Comment (prof18) [reply backend/service-import/src/jvmMain/kotlin/dev/johnoreilly/confetti/backend/import/Sessionize.kt:702]: ups

Comment (prof18) [reply backend/service-import/src/jvmMain/kotlin/dev/johnoreilly/confetti/backend/import/Sessionize.kt:702]: shall I re-push?

Comment (martinbonnin) [reply backend/service-import/src/jvmMain/kotlin/dev/johnoreilly/confetti/backend/import/Sessionize.kt:702]: Do the `curl -X post https://confetti-app.dev/update/devfestvenice2024` dance? I did it already, you should have data now.

**Pull request merged**